nat-lab/tests/config.py

```diff
@@ -41,9 +41,9 @@ def get_root_path(path: str) -> str:
 STUN_BINARY_PATH_WINDOWS = "C:/workspace/stunserver/release/stunclient.exe".replace(
     "/", "\\"
 )
-STUN_BINARY_PATH_MAC = "/var/root/stunserver/stunclient"
```

Comment:
Are these intentional changes alongside ref bump?

Reply:
+1 despite the question

Reply:
Yeah, paths changed in VMs, therefore the change in here with a bump